js_modules/dagster-ui/packages/ui-core/src/app/AppTopNav/GhostDaggy.tsx

```diff
@@ -0,0 +1,75 @@
+import {Box, Tooltip} from '@dagster-io/ui-components';
+import {Link} from 'react-router-dom';
+
+import {VersionNumber} from '../../nav/VersionNumber';
+import {WebSocketStatus} from '../WebSocketProvider';
+
+export const GhostDaggyWithTooltip = () => {
+  return (
+    <Tooltip
+      placement="bottom"
+      modifiers={{
+        offset: {enabled: true, options: {offset: [0, 12]}},
+      }}
+      content={
+        <Box flex={{direction: 'row', gap: 4}}>
+          <WebSocketStatus />
+          <VersionNumber />
+        </Box>
+      }
+      display="inline-flex"
+    >
+      <Link to="/home" style={{outline: 0, display: 'flex'}}>
+        <GhostDaggySVG />
+      </Link>
+    </Tooltip>
+  );
+};
+
+export const GhostDaggySVG = () => (
```

Comment:
Moved these out into a separate file so that we don't have to get them from `AppTopNav`.